model/relation/event/processor/ItemRemovedEventProcessor.php

```diff
@@ -38,18 +41,41 @@ public function process(Event $event): void
             throw new InvalidEventException($event);
         }
 
-        $id = $event->jsonSerialize()['itemUri'] ?? null;
+        $data = $event->jsonSerialize();
+
+        $id = $data['itemUri'] ?? null;
+        $deleteAssets = $data['deleteAssets'] ?? false;
 
         if (empty($id)) {
             throw new InvalidEventException($event, 'Missing itemUri');
         }
 
+        if ($deleteAssets) {
+            $collection = $this->getMediaRelationRepository()->getItemAssetUris($id);
+
+            $this->getLogger()->debug(
+                '====================== collection: ' . json_encode($collection)
+            );
```

Comment:
I think this information could be logged as info, in this case. Wdyt?

```suggestion
            $this->getLogger()->info(
               sprintf(
                    ' Assets "%s" removed after Item "%s" using them was removed ',
                    json_encode($collection),
                    $id
            );
```